odoo/addons/base/models/res_country.py

```diff
@@ -190,7 +190,7 @@ def write(self, vals):
 class ResCountryState(models.Model):
     _name = 'res.country.state'
     _description = "Country state"
-    _order = 'code'
+    _order = 'code, id'
```

Comment:
Nitpick: you could use 'country_id, code' (just like the unique constraint)

Reply:
I rather perfer avoiding the a left join for no reason.